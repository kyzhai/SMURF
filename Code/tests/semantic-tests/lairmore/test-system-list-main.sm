
x :: System
x = [[ (2,2)$2]]
main = [x]
