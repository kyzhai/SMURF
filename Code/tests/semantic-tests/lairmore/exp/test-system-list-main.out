Program: 
	SVardef: 
		ID: x  :: System

	[[ (2, 2)$2]]
	SMain: [x]
Global Scope: 
	ID: main  :: [System]
	ID: x  :: System
	ID: print x :: a -> a
	ID: random  :: Int

